content/en/continuous_integration/pipelines/custom.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Send custom pipelines to Datadog
+kind: documentation
+aliases:
+  - /continuous_integration/setup_pipelines/custom
+further_reading:
+    - link: "/continuous_integration/pipelines"
+      tag: "Documentation"
+      text: "Explore Pipeline Execution Results and Performance"
+    - link: "/continuous_integration/guides/pipeline_data_model"
+      tag: "Documentation"
+      text: "Learn about the Pipeline Data Model and Execution Types"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Compatibility
+
+- **Custom tags and metrics**: Attach custom tags and metrics to pipeline executions
+
+- **Manual steps**: View manually triggered pipelines
+
+- **Parameters**: Set custom parameters to pipelines
+
+- **Partial pipelines**: View partially retried and downstream pipeline executions
```

Comment:
I don't think we currently support "downstream pipeline executions" in the general implementation

```suggestion
- **Partial pipelines**: View partially retried pipelines
```